ordered_associative_task/student/src/impl.cpp

```diff
@@ -1 +1,16 @@
 #include "impl.h"
+#include <map>
+
+size_t convertMarkTo5PointSystem(size_t markIn100Point) {
+	const size_t MAX = 100;
```

Comment:
hacky way, try to solve it with such mapping:
{{0, 1}, {30, 2}, {50, 3}, {70, 4}, {90, 5}};